benchexec/tools/veriabsl.py

```diff
@@ -0,0 +1,16 @@
+#This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from . import veriabs
+ 
+class Tool(veriabs.Tool):  
+      """
+      VeriAbsL
```

Comment:
Could there be something more added about VeriAbsL, e.g., at least a link to its homepage?